src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -6754,9 +6857,12 @@ _pbs_python_event_to_request(unsigned int hook_event, hook_output_param_t *req_p
 		default:
 			log_err(PBSE_INTERNAL, __func__,
 				"unexpected hook event type");
-			return -1;
+			goto event_to_request_exit;
 	}
-	return 0;
+event_to_request_exit:
+	rc = 0;
+	hook_perf_stat_stop(perf_label, perf_action, 0);
+	return rc;
```

Comment:
does this mean this function will always return 0 (success)

Reply:
Woops...the "rc = 0" should be before the "event_to_request_exit" label. I'll fix.